### ISSUE TITLE
What does Current usage mean?

### ISSUE THREAD

**Issue by RedNo7** [masaccio/ha-kingspan-watchman-sensit#34]
Hi,

the word usage implies "L per time_period" but the unit is 'L'.  Over what time period is it calculating?  ...i.e. L/day?

Thanks!

edit: by the way, this is excellent - thanks for doing this!

**Comment by masaccio** [comment]
It is litres per day. I don’t think HA has that as a measurement option and it’s HA that generates the units. I believe that changing the description would be the only option.

**Comment by RedNo7** [comment]
Yep, just checked. The sensor device class `volume_flow_rate` only allows for:
`m³/h, ft³/min, L/min, gal/min`

so perhaps, as you wrote, something like `Daily usage` makes sense as that unit can be `L`.

**Comment by masaccio** [comment]
Would break existing integrations as the name changes so closing as won't fix.
